hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/index/hbase/TestSparkHoodieHBaseIndex.java

```diff
@@ -114,6 +114,9 @@ public class TestSparkHoodieHBaseIndex extends SparkClientFunctionalTestHarness
   @BeforeAll
   public static void init() throws Exception {
     // Initialize HbaseMiniCluster
+    System.setProperty("zookeeper.preAllocSize", "100");
+    System.setProperty("zookeeper.maxCnxns", "60");
+    System.setProperty("zookeeper.4lw.commands.whitelist", "*");
```

Comment:
why do we need these configs in this PR?

Reply:
this is caused by spark version upgrade and changed zookeeper's version.

Reply:
trying to run in CI without it. locally seems fine.

Reply:
`"zookeeper.4lw.commands.whitelist", "*"` will be required to start the test service properly with new zookeeper version used. added it back